cloud/google/gce.py

```diff
@@ -174,14 +180,15 @@
     image: debian-6
     zone: us-central1-a
     service_account_email: unique-email@developer.gserviceaccount.com
-    pem_file: /path/to/pem_file
+    credentials_file: /path/to/json_file
```

Comment:
instead of just changing it, have old and new examlpe explaining the deprecation and the library version difference